keyboards/mechlovin/hex4b/rev1/readme.md

```diff
@@ -0,0 +1,21 @@
+# hex4b
+
+![hex4b](https://i.imgur.com/lArfGgHh.jpeg)
+
+A PCB for HEX.4B 75% keyboard.
+
+* Keyboard Maintainer: [Mechlovin' Studio](https://github.com/mechlovin)
+* Hardware Supported: Hex.4B PCB, Atmega32A 
+* Hardware Availability: [Hex Keyboard](https://hexkeyboards.com/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15